genie/utils/support.py

```diff
@@ -8,20 +8,30 @@
 
 
 @frappe.whitelist()
-def create_ticket(title, description, screen_recording=None):
+def create_ticket(title, description, user, user_fullname, priority, file_attachment=None, screen_recording=None):
```

Comment:
retain argument order of function, add new args to the end and give default value